dist/node/extractors/mfcc.js

```diff
@@ -28,37 +28,13 @@ exports.default = function (args) {
 			loggedMelBands[i] += filtered[i][j];
 		}
 
-		//log each coefficient
-		loggedMelBands[i] = Math.log(loggedMelBands[i]);
+		//log each coefficient unless it's 0.
+		loggedMelBands[i] = loggedMelBands[i] > 0.00001 ? Math.log(loggedMelBands[i]) : 0;
```

Comment:
Should probably add a TODO comment to come back and see how it's done elsewhere, but it's fine for now.